app/views/kaminari/_paginator.html.haml

```diff
@@ -0,0 +1,18 @@
+-#  The container tag
```

Comment:
7 consecutive comments can be merged into one<br>Comment should have a space after the `#`